scripts/pepper.cmd

```diff
@@ -0,0 +1,5 @@
+@echo off
+set _SCRIPTDIR=%~d0
+set _SCRIPTPATH=%~p0
+
+python "%_SCRIPTDIR%%_SCRIPTPATH%pepper" %*
```

Comment:
Can you add a newline at the end of this file?

Reply:
I tried adding a newline, but it looks like Github is stripping it. As for the question of how this differs from adding the peppr script to bin/, this is for Windows users. The pepper script doesn't get added to any bin directory. Further, Windows users are used to being able to run commands without specifying the interpreter first and by default Python doesn't setup a file association that results in the execution of scripts, but in the editing of them.

Reply:
looks like the newline is good now.

Ok, so I don't have to do anything else to let the script work?

Reply:
No. Maybe just make a note somewhere pointing Windows users to it. It will otherwise work out its own install location and properly call the pepper python script that is provided by this repo.

Reply:
Can you make a note about it in the README?